scripts/common/preflights.sh

```diff
@@ -638,3 +639,13 @@ allow_remove_docker_new_install() {
           logWarn "conflicts, please uninstall Docker and re-run the install script."
      fi
 }
+
+allow_migrate_from_rook_to_openebs() {
+    if [ -z "$ROOK_VERSION" ] && [ -n "$OPENEBS_VERSION" ]; then
+        if kubectl get ns | grep -q rook-ceph; then
+            if  [ "$(echo "$OPENEBS_VERSION}" | wc -l)" -le "3" ] || [ "$OPENEBS_VERSION" == "3.2.0"]; then
```

Comment:
what about 1.x and 2.x versions?

Reply:
This code will bail when OpenEBS version is == 3.2.0 and lower.
I improve the log and just did another test to confirm:

<img width="926" alt="Screenshot 2023-03-07 at 22 01 27" src="https://user-images.githubusercontent.com/7708031/223569475-ade9bce3-1eef-44f9-8e68-9c5005c19e0e.png">

So, what do you mean with 1.x and 2.x? If they choose those then the installer will also fail.

Reply:
this looks like a typo with an extra `}` `"$(echo "$OPENEBS_VERSION}" | wc -l)" -le "3"`. I have a feeling its not bailing for the reason you think it is.

Reply:
You are shell master. Yep. You are right. I did all tests and I fixed it.